client/src/utils/cookieInfo.js

```diff
@@ -1,4 +0,0 @@
-// set a cookie
-export function setCookie(cookName, cookValue) {
-  document.cookie = cookName + "=" + cookValue + ";" + ";path=/";
-}
```

Comment:
Why do we need to remove this? My guess is that it is unused?

Reply:
Yep. It is unused.